test/integration/auth/node_test.go

```diff
@@ -230,7 +255,9 @@ func TestNodeAuthorizer(t *testing.T) {
 	expectForbidden(t, getPV(node1Client))
 	expectForbidden(t, createNode2NormalPod(nodeanonClient))
 	expectForbidden(t, createNode2MirrorPod(node1Client))
-	expectForbidden(t, deleteNode2MirrorPod(node1Client))
+	// mirror pods do not exist from the apiserver stand point
```

Comment:
I don't think a comment is needed here... the pod doesn't exist from any stand point :)